tcpb/utils.py

```diff
@@ -85,6 +131,8 @@ def job_output_to_atomic_result(
         return_result: Union[float, List[float]] = jo_dict["energy"][0]
 
     elif atomic_input.driver == "gradient":
+        # TODO: Gradient calculation should also return energy. Henry throw energy and gradient
+        # outputs to extras["qcvars"]. There should be a cleaner way.
```

Comment:
It does return energy in the `result.properties.return_energy` value.

Reply:
I am trying to follow the specification created here as much as possible so that we have a result that is interoperable with other QC packages. https://github.com/MolSSI/QCElemental/blob/ec9d754414824e129729f477e6077a14d859dc85/qcelemental/models/results.py#L543